src/cli.go

```diff
@@ -59,7 +60,8 @@ func newAPIClient(ua *userEndpointAuth, cache bool) *sourcegraph.Client {
 
 	var transport http.RoundTripper
 	if cache {
-		transport = http.RoundTripper(httpcache.NewTransport(diskcache.New("/tmp/srclib-cache")))
+		tempdir := filepath.Join(os.TempDir(), "srclib-cache")
```

Comment:
No problem. Please ping me once you've done with the comments I posted here and in https://github.com/sourcegraph/srclib-go/pull/38. I saw there were a few instances of `path.Join` being used where `filepath.Join` should be used instead.

Then I'll review all of your changes again. Thanks!